src/main/java/lotto/domain/LottoRank.java

```diff
@@ -0,0 +1,55 @@
+package lotto.domain;
+
+import lotto.util.Money;
+import lotto.util.number.MultiplicationNumber;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public enum LottoRank {
+    LAST_PLACE  (new Money(0)),
+    FOUR_PLACE  (3, new Money(5000)),
```

Comment:
```suggestion
    FOUR_PLACE  (3, new Money(5_000)),
```
위와 같이 언더스코어를 통해 단위를 표기해주면 가독성을 향상시킬 수 있습니다ㅎㅎ